### ISSUE TITLE
Pre-training the ViT-tiny encoder

### ISSUE THREAD

**Issue by goutamyg** [RobinWu218/SimDINO#41]
Hi @RobinWu218,

I tried pre-training a ViT-tiny encoder on ImageNet1k dataset in SimDINOv2 style. All the hyperparameters were kept the same as **_simdinov2/configs/simdino_config.yaml_**. The batch size was set to 512-per-gpu that fits on two RTX A6000 GPUs I have (total batch size: 1024). 

I compared my training plots (below) with the one you shared for ViTb-16 (here https://github.com/RobinWu218/SimDINO/issues/13) and notice the loss values for the two encoders have a similar trend. 

![Image](https://github.com/user-attachments/assets/ae27ef4b-a6b2-454e-a3dd-ab5b0c401b53)

However: 

(i) While the _dino_mcr_loss_ and _total_loss_ are decreasing, the patch-based _ibot_comp_loss_ increases during the later part of the training. Will that affect the model performance, esp. on dense prediction tasks?

(ii) The _dino_mcr_expa_loss_ is significantly high compared to ViTb-16 and has visible fluctuations. Is it normal or does it indicate lack of diversity in learned features? 

Any help towards interpreting the loss values and possibly fix these issues will be helpful. Thank you! 

-Goutam

**Comment by RobinWu218** [comment]
(i) The ibot_comp_loss stays in similar ranges as we observed in our own training logs. We are not too sure about its effect on downstream tasks. A way to assess this would be to evaluate tasks on checkpoints at various stages of training.

(ii) The fluctuation is actually pretty small in the magnitude of the coding rate values. The difference likely comes from the batch size and is normal behavior. As long as the value is stable, and you see the compression_loss improve, you should be good.
